cubeseed_login/src/pages/login_page/login-page.tsx

```diff
@@ -11,46 +11,44 @@
 
 // export default LoginPage;
 
-
 import React, { useState } from "react";
 import { useRouter } from "next/router";
 import Link from "next/link";
-import Image from 'next/image';
-import { AiFillGoogleCircle } from "react-icons/ai";
-import { BsFacebook } from "react-icons/bs";
-import { AiFillTwitterCircle } from "react-icons/ai";
+import Image from "next/image";
+// import { AiFillGoogleCircle } from "react-icons/ai";
+// import { BsFacebook } from "react-icons/bs";
+// import { AiFillTwitterCircle } from "react-icons/ai";
 import loginoptions from "@/assets/loginoptions.png";
 import logo from "@/assets/cubeseed.png";
 import styles from "@/styles/loginpage.module.scss";
 
-
 interface LoginFormProps {
   onSubmit: (email: string, password: string, rememberMe: boolean) => void;
   onForgotPassword: () => void;
 }
 
 const LoginPage: React.FC<LoginFormProps> = ({
   onSubmit,
-  onForgotPassword
+  onForgotPassword,
 }) => {
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
   const [rememberMe, setRememberMe] = useState(false);
-  const [emailError, setEmailError] = useState('');
-  const [passwordError, setPasswordError] = useState('');
+  const [emailError, setEmailError] = useState("");
+  const [passwordError, setPasswordError] = useState("");
   const router = useRouter();
 
   const handleLogin = () => {
-    if (email === '') {
```

Comment:
i suggest using a lib for forms, https://react-hook-form.com/ is a good one.

Reply:
i did not work on this